redbot/cogs/audio/audio.py

```diff
@@ -570,7 +575,8 @@ def _restart_connect(self):
         """Toggle the domain restriction on Audio.
 
         When toggled off, users will be able to play songs from non-commercial websites and links.
-        When toggled on, users are restricted to YouTube, SoundCloud, Mixer, Vimeo, Twitch, and Bandcamp links."""
+        When toggled on, users are restricted to YouTube, SoundCloud, \
```

Comment:
why? (and also, dont use `\` line continuations here)

Reply:
Yeah, this'll get rendered like this in the help command:
```
When toggled on, users are restricted to YouTube, SoundCloud,         Mixer, Vimeo, Twitch, and Bandcamp links.
```

I always get slightly annoyed with how docstrings end up looking in the help command - sticking to the 99-char line length but also considering where the line breaks will appear in the help embed is frustrating.

My solution would be to rewrap/unwrap each paragraph in the help formatter. Code-block help formatter should rewrap, whilst embed help formatter should unwrap. This means a single newline would usually treated as a space (we can make exceptions, possibly for numbered/unordered lists) and double newlines are preserved. Still not sure about this though.

Reply:
Let's keep the docstring free of line continuations here.

I'll open an issue + handle this case for 3.2 in how we handle help.